### PR TITLE
Splashpage

### PR DESCRIPTION
**Pull request by thesauri**
Added the CSS framework Foundation.

The menu items (Om Teknoloföreningen, medlemsportal m.m.) are hidden under a hamburger menu on mobile devices. The TF logo is placed under the menu bar with the menu underneath (which is loaded from the API). The links to Täffä and Täffä AB are placed as columns in a row below the menu. They are supposed to collapse to separate rows on small devices, but this is not implemented yet. The language selection links are found at the very bottom.

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -1,48 +1,70 @@
...
-    
-    $TaffaAPI = new TaffaAPI(@$_GET['lang']);
+include_once './TaffaAPI.class.php';
+$TaffaAPI = new TaffaAPI(@$_GET['lang']);
 ?>
 <!DOCTYPE html>
 <html>
-  <head>
+<head>
+  <!-- TEKNOLOGRÖD: #b20738 -->
```
Comment (oehnstro) [inline index.php]: This should be in the style.css file.

File: index.php [hunk 2]
```diff
@@ -1,48 +1,70 @@
...
   <title>Hej TF</title>
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width, initial-scale=1">
+
+  <link rel="stylesheet" href="css/foundation.css">
+  <link rel="stylesheet" href="css/normalize.css">
+
+  <link href='https://fonts.googleapis.com/css?family=Old+Standard+TT:400,400italic,700' rel='stylesheet' type='text/css'>
```
Comment (oehnstro) [inline index.php]: We're not using this font anymore. Should be deleted.

File: index.php [hunk 3]
```diff
@@ -1,48 +1,70 @@
...
+        <li><a href="#">Medlemsportal</a></li>
+        <li><a href="#">Arbiturienter</a></li>
+        <li><a href="#">Alumner</a></li>
+        <li><a href="#">Samarbete</a></li>
+      </ul>
+    </section>
+  </nav>
+  <div class="row">
+    <div class="small-6 small-centered column">
+      <img src="assets/tf_natside_logon-08.svg" alt="Teknologföreningens logga" style="max-height: 270px">
```
Comment (oehnstro) [inline index.php]: Inline style should be moved to style.css

File: style.css [hunk 4]
```diff
@@ -12,20 +11,32 @@
...
+	url('fonts/Montserrat-Light.woff') format('woff'),
+	url('fonts/Montserrat-Light.ttf')  format('truetype'),
 	url('fonts/Montserrat-Light.svg#Montserrat-Light') format('svg');
   font-weight: normal;
   font-style: normal;
 }
 
-body {background-color: red; font: 36px "Montserrat-Light";}
+body {
+  font: 36px "Montserrat-Light";
```
Comment (oehnstro) [inline style.css]: Add backup font family for backwards compatibility. e.g.
`font-family: 'Montserrat-Light', sans-serif;`

File: style.css [hunk 5]
```diff
@@ -12,20 +11,32 @@
...
+}
 
 h1 {
   font-family: "Montserrat-Bold";
```
Comment (oehnstro) [inline style.css]: Backup font family, see previous.

File: style.css [hunk 6]
```diff
@@ -12,20 +11,32 @@
...
+footer ul {
+  text-align: center;
+}
+
+footer ul li {
+  display: inline;
+}
 
+.top-bar-section ul li > a {
+  font-family: "Montserrat-Light"
```
Comment (oehnstro) [inline style.css]: Backup font family, see previous.

**Comment by oehnstro** [comment]
Unused foundation javascript libraries should be removed.

Overall good work! :+1:  Small fixes here and there required.

**Comment by thesauri** [comment]
Fixed the above-mentioned problems. Only foundation.min.js and jquery.js are included now.

**Pull request merged**